encoder/src/main/java/com/pedro/encoder/input/gl/render/ManagerRender.java

```diff
@@ -76,6 +76,10 @@ public void release() {
     screenRender.release();
   }
 
+  public boolean isPortraitStart() {
```

Comment:
@SouOka 
isPortraitのままではまずいでしょうか？
screenRenderから取得している他のgetterも名前はそのままです
```suggestion
  public boolean isPortrait() {
```

Reply:
@mn-D128 
外部から見た時にisPortraitのままだと、今がportraitって意味みたいでわかりにくいかな〜っと思って変えたんですが、そのままの方がよければそちらに合わせます

Reply:
@SouOka 
内部メンバ変数もisPortraitなので、isPortraitでいきましょう

Reply:
@mn-D128 
d82a52f で対応